Documentation/index.md

```diff
@@ -1,57 +1,81 @@
 ---
-title: Azure Remote Rendering asset tool user documentation
-description: Azure Remote Rendering Asset Tool Documentation for the user
-author: mafranc
-ms.author: mafranc
-ms.date: 03/23/2020
+title: Azure Remote Rendering Toolkit User Documentation
+description: User documentation for the Azure Remote Rendering Toolkit
+author: jakras
+ms.author: jakras
+ms.date: 12/23/2021
 ms.topic: article
 ---
 
-# Azure Remote Rendering asset tool (ARRT) user documentation
+# Azure Remote Rendering Toolkit (ARRT) User Documentation
 
 ![ARRT material editing view](media/ARRT.png)
 
-ARRT (Azure Remote Rendering asset tool) is an application using Azure Remote Rendering service (ARR) to upload, visualize, and edit 3D models.
+Azure Remote Rendering Toolkit (ARRT) demonstrates the functionality of [Azure Remote Rendering (ARR)](https://docs.microsoft.com/azure/remote-rendering). ARRT functions as a sample application as reference for how to integrate remote rendering into your own product. However, it can also be a useful tool to get basic tasks done, like uploading a source asset into Azure Storage or converting a model for remote rendering. As such it makes it much easier to get started with remote rendering.
 
-ARRT can be used by technical artists or contents creators to:
+## List of Features
 
-* convert 3D models (.fbx/.gtlf/.glb) to the ARR internal format (.arrAsset)
-* start an ARR session to load
-* render the model remotely
-* modify its materials
+These features are currently available in ARRT:
 
-For more information on the input formats supported by ARR, see [Supported source formats](https://docs.microsoft.com/en-gb/azure/remote-rendering/how-tos/conversion/model-conversion#supported-source-formats)
+1. [Upload files](upload.md) into an Azure Storage account.
```

Comment:
docs policies state that you should use numerated lists only in case users need to follow steps in that order. Not for enumerating features